specification/columns/commitmentdiscountquantity.md

```diff
@@ -10,17 +10,17 @@ The CommitmentDiscountQuantity column adheres to the following requirements:
 
 * CommitmentDiscountQuantity MUST be present in a FOCUS dataset when the provider supports *commitment discounts*.
 * CommitmentDiscountQuantity MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements.
-* CommitmentDiscountQuantity MAY be negative if [*ChargeClass*](#chargeclass) is "Correction".
+* CommitmentDiscountQuantity MAY be null or any valid decimal value if *ChargeClass* is "Correction".
```

Comment:
even in case of the correction, any valid decimal value should be allowed only if CommitmentDiscountId is not null

Reply:
Applied this ^^